src/scss/_themes.scss

```diff
@@ -1,47 +1,56 @@
 @mixin oSubsCardB2B {
 	.o-subs-card__top {
-		@include oColorsFor('o-subs-card-b2b');
+		background: oColorsByName('slate');
+		color: oColorsByName('white');
 	}
 
 	.o-subs-card__copy-benefits {
-		@include oColorsFor('o-subs-card-b2b');
+		background: oColorsByName('slate');
+		color: oColorsByName('white');
 	}
 
 	.o-subs-card__copy-title:first-child {
-		@include oTypographyMargin($bottom: 1);
-		@include oTypographyPadding($top: 6);
+		margin-bottom: oSpacingByName('s1');
+		padding-top: oSpacingByName('s6');
 		padding-right: $o-subs-card-horizontal-padding;
 		padding-left: $o-subs-card-horizontal-padding;
 	}
 
 	.o-subs-card__select-button {
-		@include oButtonsTheme(inverse);
+		@include oButtonsContent($opts: (
+			'type': 'secondary',
+			'size': 'big',
+			'theme': ('inverse')
```

Comment:
As a modifier, I think we can reduce the css bundle size a little here by passing `$include-base-styles: false` as a second argument. Which means we don't need to specify `'size': 'big'` either, as it doesn't need to override the default style

Reply:
Tried this suggestion but we then realised I don't have the base styles in this anywhere.